maro/cli/grass/lib/agents/master_agent.py

```diff
@@ -8,137 +8,154 @@
+    set_containers_details, set_job_details
 )
 
 logger = logging.getLogger(__name__)
 
-START_CONTAINER_COMMAND = """\
-ssh -o StrictHostKeyChecking=no {admin_username}@{node_hostname} \
-docker run \
--it -d \
---cpus {cpu} \
--m {memory} \
---name {container_name} \
---network host \
---log-driver=fluentd \
---log-opt tag=maro.job_id.{job_id}.container_name.{container_name} \
---log-opt fluentd-address={master_hostname}:{fluentd_port} \
---label required_cpu={cpu} \
---label required_memory={memory} \
---label required_gpu={gpu} \
--v {mount_source}:{mount_target} \
-{environment_parameters} \
-{image_name} {command}\
-"""
-
-START_CONTAINER_WITH_GPU_COMMAND = """\
-ssh -o StrictHostKeyChecking=no {admin_username}@{node_hostname} \
-docker run \
--it -d \
---cpus {cpu} \
--m {memory} \
---gpus {gpu} \
---name {container_name} \
---network host \
---log-driver=fluentd \
---log-opt tag=maro.job_id.{job_id}.container_name.{container_name} \
---log-opt fluentd-address={master_hostname}:{fluentd_port} \
---label required_cpu={cpu} \
---label required_memory={memory} \
---label required_gpu={gpu} \
--v {mount_source}:{mount_target} \
-{environment_parameters} \
-{image_name} {command}\
-"""
-
-REMOVE_CONTAINER_COMMAND = """\
-ssh -o StrictHostKeyChecking=no {admin_username}@{node_hostname} \
-docker rm {containers}\
-"""
-
-STOP_CONTAINER_COMMAND = """\
-ssh -o StrictHostKeyChecking=no {admin_username}@{node_hostname} \
-docker stop {containers}\
-"""
-
-METRIC_TO_INDEX = {
-    'cpu': 1,
-    'memory': 2,
-    'gpu': 3
+START_CONTAINER_COMMAND = (
+    "ssh -o StrictHostKeyChecking=no {admin_username}@{node_hostname} "
+    "docker run "
+    "-it -d "
+    "--cpus {cpu} "
+    "-m {memory} "
+    "--name {container_name} "
+    "--network host "
+    "--log-driver=fluentd "
+    "--log-opt tag=maro.job_id.{job_id}.container_name.{container_name} "
+    "--log-opt fluentd-address={master_hostname}:{fluentd_port} "
+    "-v {mount_source}:{mount_target} "
+    "{environment_parameters} {labels} "
+    "{image_name} {command}"
+)
+
+START_CONTAINER_WITH_GPU_COMMAND = (
+    "ssh -o StrictHostKeyChecking=no {admin_username}@{node_hostname} "
+    "docker run "
+    "-it -d "
+    "--cpus {cpu} "
+    "-m {memory} "
+    "--gpus {gpu} "
+    "--name {container_name} "
+    "--network host "
+    "--log-driver=fluentd "
+    "--log-opt tag=maro.job_id.{job_id}.container_name.{container_name} "
+    "--log-opt fluentd-address={master_hostname}:{fluentd_port} "
+    "-v {mount_source}:{mount_target} "
+    "{environment_parameters} {labels} "
+    "{image_name} {command}"
+)
+
+REMOVE_CONTAINER_COMMAND = (
+    "ssh -o StrictHostKeyChecking=no {admin_username}@{node_hostname} "
+    "docker rm -f {containers}"
+)
+
+STOP_CONTAINER_COMMAND = (
+    "ssh -o StrictHostKeyChecking=no {admin_username}@{node_hostname} "
+    "docker stop {containers}"
+)
+
+AVAILABLE_METRICS = {
+    "cpu",
+    "memory",
+    "gpu"
+}
+
+ERROR_CODE_FOR_NOT_RESTART = 64
```

Comment:
Use this: from maro.utils.exit_code import KILL_ALL_EXIT_CODE(65), NON_RESTART_EXIT_CODE(64)